periph_sw/nRF52_RGB_Strip/eeprom_handler.h

```diff
@@ -34,10 +43,14 @@ class EEPROM_Handler
     {
     }
 
-    void static load(Settings &settings)
+    void configure(void)
```

Comment:
This function should also be static, right?

Reply:
Could be yes, will set it static.